pkg/golinters/govet_test.go

```diff
@@ -18,7 +18,7 @@ import (
 
 func TestGovet(t *testing.T) {
 	// Checking that every default analyzer is in "all analyzers" list.
-	checkList := append(defaultAnalyzers,
+	checkList := append(defaultAnalyzers, //nolint:gocritic // Build a temporary list of analyzers.
```

Comment:
Depending on the code style used in this project, there are multiple solutions.
Given that the original code looks like this:
```go
	checkList := append(defaultAnalyzers,
		shadow.Analyzer, // special case, used in analyzersFromConfig
	)
```
We can do something from the options below.
```go
checkList := defaultAnalyzers
checkList = append(checkList, shadow.Analyzer)
```
```go
var checkList []*analysis.Analyzer
checkList = append(checkList, defaultAnalyzers...)
checkList = append(checkList, shadow.Analyzer)
```
I think there are more alternatives available, but these should suffice.